Chapter11/das_node.erl

```diff
@@ -0,0 +1,53 @@
+-module(das_node).
+-export([create/0, address_loop/0]).
+
+-include("nodes.hrl").
+
+-vsn(1.0).
+
+% Exercise 11-1: Distributed Associative Store (Main Nodes)
+
+create() -> register(address_loop, spawn(das_node, address_loop, [])).
+
+address_loop() ->
+  receive
+    {link, Node} ->
+      net_kernel:connect(Node),
+      monitor_node(Node, true),
+      {address_loop, Node} ! {link_node, node()},
+      address_loop();
+
+    {link_node, Node} ->
+      % net_kernel:connect(Node),
+      monitor_node(Node, true),
```

Comment:
It doesn't look like monitor is used.  I don't see that process can receive `nodedown` messages and handle them.